.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Ofc we could name every tag starting with `v` but it's not specified anywhere so you should just use `*` or just simple `on: push: tags` or add appropriate entry for `CONTRIBUTION.md`.